stdlib/REPL/src/TerminalMenus/AbstractMenu.jl

```diff
@@ -362,10 +368,15 @@ down_arrow(c::AbstractConfig) = down_arrow(c.config)
 down_arrow(c::Config) = c.down_arrow
 down_arrow(::AbstractMenu) = CONFIG[:down_arrow]
 
-print_arrow(buf, ::ConfiguredMenu, c::Char) = print(buf, c, "  ")
+updown_arrow(m::ConfiguredMenu) = updown_arrow(m.config)
+updown_arrow(m::AbstractConfig) = updown_arrow(c.config)
```

Comment:
```suggestion
updown_arrow(c::AbstractConfig) = updown_arrow(c.config)
```
Maybe a sign that there is a missing test?

Reply:
I believe now it should get covered by a test with `charset=:ascii` added to trigger ConfiguredMenu instead of legacy Dict-based menu.